tools/introspection/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>iceoryx_introspection</name>
+    <version>2.90.0</version>
+    <description>Eclipse iceoryx inter-process-communication (IPC) middleware introspection client</description>
+    <maintainer email="iceoryx-oss-support@apex.ai">Eclipse Foundation, Inc.</maintainer>
```

Comment:
Shouldn't this go to the eclipse iceoryx mailing list?

Reply:
@elBoberido You mean [iceoryx-dev@eclipse.org](mailto:iceoryx-dev@eclipse.org)? Not sure as everyone has to subscribe to the list to send a message. Would then a ROS sending address be a subscribed member to send a message? I never had the impression that this mailing list was suitable for sending a message to the core maintainers. But I see that it would make sense to have a better way to contact core maintainers across companies

Reply:
That's a good point @budrus We had several occasions so far where people sent emails to the iceoryx-dev@eclipse.org mailing list without having subscribed before and the messages just disappeared and never arrived.

Let's talk about how we can address this point in the next developer meetup.

Reply:
Those mailing lists are usually moderated to prevent spam. Emails from non subscribed addresses need to be checked by the moderator(s) and then unlocked. Do you know who the moderators are? Maybe these mails are sent to our old Bosch accounts :smile:

Reply:
Okay, Eclipse handles this differently than other organizations and one needs to be indeed subscribed. That's a pity and we need to find a way. We need to check if ROS mailing lists allow mails from unsubscribed sender.

Reply:
@elBoberido 

> We need to check if ROS mailing lists allow mails from unsubscribed sender.

I doubt that any sane mailing list will allow this as this will very likely result in a lot of spam.

Reply:
Are you suggesting that the KDE mailing list maintainer are insane :zany_face: